it/uk/gov/hmrc/incometaxproperty/connectors/IncomeTaxWireMockServer.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incometaxproperty.connectors
+
+import com.github.tomakehurst.wiremock
+import com.github.tomakehurst.wiremock.client.WireMock
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig
+import play.api.Application
+import play.api.inject.guice.GuiceApplicationBuilder
+
+trait IncomeTaxWireMockServer {
```

Comment:
we have this class already in `uk.gov.hmrc.incometaxproperty.support.helpers.WireMockServer` so we can delete this

Reply:
Poor naming but removed mine. We should not adopt a publically acceptable name like String, Int etc as industrially they mean something different. Guess we can add it as tech debt